lib/ansible/modules/windows/win_user.ps1

```diff
@@ -219,9 +219,9 @@ If ($state -eq 'present') {
             $user_obj.SetInfo()
         }
         If ($null -ne $groups) {
-            [string[]]$current_groups = $user_obj.Groups() | ForEach { $_.GetType().InvokeMember("Name", "GetProperty", $null, $_, $null) }
+            [string[]]$current_groups = $user_obj.Groups() | ForEach-Object { $_.GetType().InvokeMember("Name", "GetProperty", $null, $_, $null) }
             If (($groups_action -eq "remove") -or ($groups_action -eq "replace")) {
-                ForEach ($grp in $current_groups) {
+                Foreach ($grp in $current_groups) {
```

Comment:
Try and limit cosmetic changes like work casing to a minimum. It changes the git attribution for sections which is invaluable when tracking down issues.